service.go

```diff
@@ -276,3 +276,7 @@ func NewService(serviceName string, serviceVersion string, confProfile string, c
 func RunningService() *Service {
 	return svc
 }
+
```

Comment:
godoc is needed for public API
// DriverConfigs retrieves the driver specific configuration

Reply:
Added